.travis.yml

```diff
@@ -22,24 +24,24 @@ env:
     - CATKIN_PARALLEL_TEST_JOBS="-p1"
     - NOT_TEST_INSTALL=true 
   matrix:
-    - ROS_DISTRO=hydro    USE_DEB=true
-    - ROS_DISTRO=hydro    USE_DEB=false EXTRA_DEB="ros-hydro-convex-decomposition ros-hydro-ivcon" BUILD_PKGS="jsk_pr2_calibration jsk_robot_startup pr2_base_trajectory_action jsk_pr2_startup jsk_pr2_desktop"
-    - ROS_DISTRO=indigo   USE_DEB=true
-    - ROS_DISTRO=indigo   USE_DEB=false EXTRA_DEB="ros-indigo-convex-decomposition ros-indigo-ivcon"
-    - ROS_DISTRO=jade     USE_DEB=true
-    - ROS_DISTRO=jade     USE_DEB=false EXTRA_DEB="ros-jade-convex-decomposition ros-jade-ivcon"
-    - ROS_DISTRO=kinetic  USE_DEB=true
-    - ROS_DISTRO=kinetic  USE_DEB=false EXTRA_DEB="ros-kinetic-convex-decomposition ros-kinetic-ivcon"
+    - ROS_DISTRO=hydro   USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=hydro   USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-hydro-convex-decomposition ros-hydro-ivcon" BUILD_PKGS="jsk_pr2_calibration jsk_robot_startup pr2_base_trajectory_action jsk_pr2_startup jsk_pr2_desktop"
+    - ROS_DISTRO=indigo  USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=indigo  USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-indigo-convex-decomposition ros-indigo-ivcon"
+    - ROS_DISTRO=jade    USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=jade    USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-jade-convex-decomposition ros-jade-ivcon"
+    - ROS_DISTRO=kinetic USE_TRAVIS=true USE_DEB=true
+    - ROS_DISTRO=kinetic USE_TRAVIS=true USE_DEB=false EXTRA_DEB="ros-kinetic-convex-decomposition ros-kinetic-ivcon"
```

Comment:
you can add `USE_TRAVIES` at `global` section

Reply:
thank you, i updated